library/Erfurt/Store/Adapter/EfZendDb.php

```diff
@@ -282,25 +253,11 @@ public function addMultipleStatements($graphUri, array $statementsArray, array $
 
```

Comment:
The `$data` array in question is only used in those two lines:

                         $dType   = substr((string)$data['od'], 0, 128) . $dTypeHash;
                         $data['od_r'] = $dtRef;

where it actually can't have any effect since it was not set before.